### PR TITLE
Nicer installer requirements page.

### PR DESCRIPTION
**Pull request by freshface**
## Type

- Enhancement

## Resolves the following issues

Installer with requirements is nicer.

Before:

screen shot 2017-02-28 at 14 35 25

After: 

screen shot 2017-02-28 at 14 35 28

### CODE REVIEW

File: src/ForkCMS/Bundle/InstallerBundle/Resources/views/Installer/step1.html.twig [hunk 1]
```diff
@@ -36,59 +36,59 @@
               <h3>PHP Extensions</h3>
               <div class="list-group">
                 <div class="list-group-item list-group-item-{{ checker.errors.extensionCURL }}">
-                  <h4 class="list-group-item-heading">cURL</h4>
+                  <h3 class="list-group-item-heading">cURL</h3>
```
Comment (carakas) [inline src/ForkCMS/Bundle/InstallerBundle/Resources/views/Installer/step1.html.twig]: these are h4's since they fall under the h3 of PHP Extensions

Comment (StijnVrolijk) [inline src/ForkCMS/Bundle/InstallerBundle/Resources/views/Installer/step1.html.twig]: I agree

**Comment by StijnVrolijk** [comment]
I like it, just change the h3's

File: src/ForkCMS/Bundle/InstallerBundle/Resources/views/Installer/step1.html.twig (hunk 1 above)
Comment (freshface) [inline src/ForkCMS/Bundle/InstallerBundle/Resources/views/Installer/step1.html.twig]: I like them smaller, but the h3 is styled smaller then the h4 which is not correct.
I will fix that also.

**Pull request merged**